src/_base.scss

```diff
@@ -32,6 +32,13 @@
   background-color: var(--bg-warning);
 }
 
+%imageMeaningfulAlt {
```

Comment:
Maybe we could make this a little more generic and/or use a mixin if needed (similar to the content mixin) so we can reuse this for other void elements?

Reply:
Having a mixing that generates the image is a good idea. I'll add it to this PR, but having other elements using it, should be their own changes and not part of this one.

Reply:
I added a more generic way of creating the image. I had to move the colors to an SCSS file so they could be read in the image generation function (as we cannot use CSS variables in the inline SVG) and added an example on how it looks for the embed inside a button:

![screenshot](https://i.stack.imgur.com/4t3UM.png)

But I didn't do the rest of the examples because I wanted to keep this PR within its original scope.